app/_src/kubernetes-ingress-controller/reference/faq/router.md

```diff
@@ -4,3 +4,9 @@ type: reference
 purpose: |
   What is the router flavor? What's the default? What difference does it make? (Gateway API only possible with expressions).
 ---
+
+{{ site.kic_product_name }} supports the `traditional`, `traditional_compatible` and `expressions` {{ site.base_gateway }} routers. Set `gateway.env.router_flavor` in your `values.yaml` when installing with Helm to configure the router used.
+
+When using Gateway API resources we recommend using the `expressions` router as it has support for advanced features such as query string matching that `traditional` and `traditional_compatible` do not support.
```

Comment:
```suggestion
When using Gateway API resources we recommend using the `expressions` router because it supports advanced features such as query string matching. The `traditional` and `traditional_compatible` routers do not support query string matching feature.
```